app/views/projects/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Project Show Page</h1>
+
+<p><%= @project.name%></p>
+<p>Material: <%= @project.material%></p>
+<p>Challenge Theme: <%= @theme%></p>
+<p>Number of Contestants: <%= @count%></p>
+<p>Average Contestant Experience: <%= @exp%></p>
+
```

Comment:
See my message in your controller. In here is where we would want to take the lift off of our controller and just call the things in the view. So for example on line 5, we would want to just call
```ruby
@project.challenge_theme
```
Same idea for lines 6 and 7.